### DISCUSSION TITLE
user id에 대한 유효성 체크 유무

### DISCUSSION THREAD

**Discussion by Kimhan-nah** [GhostPangPang/GhostPong#88] [Q&A] [answered]
### Overview
- user Id에 대한 유효성 체크 기준

### Discussion
- 현재 user id가 DB에서 find해서 존재하는 user인지에 대한 유효성 검사를 모든 곳에서 하고 있음
- 유저 본인의 id는 jwt 토큰 체크를 할 건데 이것도 DB에 존재하는 유저인지에 대한 유효성 검사를 할 것인지 (jwt로 가능하지 않을까 하는 의견)
- 유저 본인 Id인 경우 예시 : 내 정보 수정

### Participants
@Kimhan-nah @jis-kim @anso33

**Comment by jis-kim** [comment, marked as answer]
친구 신청에서도 본인 id 가 필요한데, `friendship` 테이블에서 만약 본인 id 가 유효하지 않다면 500 에러가 나지만 현재 auth 로직으로 이미 유효성 체크가 되었다고 판단해서 따로 DB 에서 select 후 확인하지는 않습니다. 
내 정보 수정에서도 바로 `insert` 로직을 해도 상관없다고 생각합니다!
결론 : 내 id 의 모든 유효성 검사는 필요 없다.
